### DISCUSSION TITLE
suggestions on how to display multiple notifications at the same time to user?

### DISCUSSION THREAD

**Discussion by 321github123** [RocketCommunicationsInc/astro#129] [Q&A] [answered]
Hi,

I was wondering if anyone had suggestions on how to display multiple messages to users at the same time e.g. "Add pressure to tires." and "Change oil.".

According to guidelines [here](https://www.astrouxds.com/components/notification-banner/),  notification banners should not stack.  It seems we should only display only one notification banner at a time? Hence a notification banner might not be the correct tool to display multiple notification messages to the user at at time? 

Thanks for your help.

**Comment by markacianfrani** [comment, marked as answer]
Our recommended approach would be something like:

> Using a Log-like table for a list of notifications and a Notifications Monitoring Icon in the Global Status Bar with a count. If it still needs to be raised up in terms of attention, then the notification banner could go up with a message that urgent notifications are available in the list or then the notifications could show up as banners in order of criticality.
